docs/Cloud_integrations/LoRaWAN/LoRagateway.md

```diff
@@ -0,0 +1,101 @@
+# LoRaWAN Gateway
+
+## Overview
+
+A LoRaWAN gateway is a radio module that functions as a communication device between end devices and a LoRaWAN network server (LNS) within a LoRa network.
+
+A LoRa network is a low-power, wide-area network (LPWAN) designed for connecting devices over long distances with low power consumption. This network is perfect for IoT sensor networks and asset-tracking applications.
+
+Once a gateway receives data from an end-device, it forwards it to the LNS which processes and delivers the data to the appropriate application server.
+
+## Prerequisites
+
+You need an active [The Things Stack Cloud](https://www.thethingsindustries.com/stack/plans/) account. For more information, read The Things Stack Cloud [documentation](https://www.thethingsindustries.com/docs/the-things-stack/cloud/).
+
+## Integration with LoRaWAN Network Server
+
+To connect LoRa gateway devices with Coiote, you need to set up integration with a LoRaWAN Network Server - The Things Stack. To set up the integration:
```

Comment:
```suggestion
To connect LoRaWAN gateway devices with Coiote, you need to set up integration with a LoRaWAN Network Server - The Things Stack. To set up the integration:
```

Please, see my previous comments. We can also discuss this outside this MR.

Reply:
Also, The Things Stack or The Things Network is not the only LoRaWAN Network Server. We, as a company, support TTS/TTN network servers (for now). Maybe we should clarify that?